{
  "id": "CVE-2011-4103",
  "assigner": "secalert@redhat.com",
  "Published": "2014-10-27T00:55:23.28Z",
  "Modified": "2014-12-18T14:35:53.143Z",
  "last-modified": "2014-12-18T14:35:53.143Z",
  "summary": "emitters.py in Django Piston before 0.2.3 and 0.2.x before 0.2.2.1 does not properly deserialize YAML data, which allows remote attackers to execute arbitrary Python code via vectors related to the yaml.load method.",
  "access": {
    "authentication": "NONE",
    "complexity": "LOW",
    "vector": "NETWORK"
  },
  "impact": {
    "availability": "PARTIAL",
    "confidentiality": "PARTIAL",
    "integrity": "PARTIAL"
  },
  "cvss": 7.5,
  "exploitabilityScore": 10.0,
  "impactScore": 6.4,
  "cvss-vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P",
  "references": [
    "http://www.debian.org/security/2011/dsa-2344",
    "http://www.openwall.com/lists/oss-security/2011/11/01/10",
    "https://bitbucket.org/jespern/django-piston/commits/91bdaec89543/",
    "https://bugzilla.redhat.com/show_bug.cgi?id=750658",
    "https://www.djangoproject.com/weblog/2011/nov/01/piston-and-tastypie-security-releases/"
  ],
  "vulnerable_configuration": [
    "cpe:2.3:a:djangoproject:piston:0.2.2.0:*:*:*:*:*:*:*"
  ],
  "vulnerable_product": [
    "cpe:2.3:a:djangoproject:piston:0.2.2.0:*:*:*:*:*:*:*"
  ],
  "vendors": [
    "djangoproject"
  ],
  "products": [
    "piston"
  ],
  "vulnerable_product_stems": [
    "cpe:2.3:a:djangoproject:piston"
  ],
  "vulnerable_configuration_stems": [
    "cpe:2.3:a:djangoproject:piston"
  ],
  "cwe": "CWE-20",
  "vulnerable_configuration_cpe_2_2": []
}